tedana/tests/test_utils.py

```diff
@@ -133,8 +133,7 @@ def test_make_adaptive_mask():
     mask, masksum = utils.make_adaptive_mask(data, mask=pjoin(datadir,
                                                               'mask.nii.gz'),
                                              minimum=False, getsum=True)
-    assert np.allclose(mask, nib.load(pjoin(datadir,
-                                            'mask.nii.gz')).get_data().flatten())
+    assert np.allclose(mask, masksum.astype(bool))
```

Comment:
Our test is failing, here, since we're now hitting this "bad voxel" condition, I suppose ? This is probably actually fine for now since it covers more lines of code, but we could add a "#TODO" here about testing with a mask that does not include bad voxels.

Reply:
Yeah, a lot of voxels in the mask had bad data, so the mask file didn't match up with the adaptive mask. I'll add a TODO.